src/main/java/pl/jakubowskir/timetable/trainer/controller/TrainerDto.java

```diff
@@ -0,0 +1,7 @@
+package pl.jakubowskir.timetable.trainer.controller;
+
+public class TrainerDto {
+    String Name;
+    String Surname;
+    Long id;
```

Comment:
Id jest generowane automatycznie, nie musisz dodawać tego do Dto

Reply:
@Gorgios mam to usunąć czy zostawić tak jak jest ?

Reply:
Usuń, Id będzie dodane automatycznie przy dodawaniu do bazy.